### PR TITLE
arash/default to volatility indices on weekends

### PR DESCRIPTION
**Pull request by arash-binary**

### CODE REVIEW

File: src/javascript/app_2/Stores/Modules/Trading/trade-store.js [hunk 1]
```diff
@@ -199,6 +200,36 @@ export default class TradeStore extends BaseStore {
                 });
             }));
         }
+
+        // Change default symbol based on client's favorites and symbol exchange_is_open status
+        const current_active_symbol = active_symbols.active_symbols.find(s => s.symbol === this.symbol);
+        const client_favorite_store = JSON.parse(LocalStore.get('cq-favorites'))['chartTitle&Comparison'];
+
+        // Have not favorites
+        if (client_favorite_store.length === 0) {
```
[the remark this answers is not in the document]
Comment (arash-binary) [reply src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: This is not boolean variable so, I think it does not make a sense to name it like the boolean type.

[the remark this answers is not in the document]
Comment (arash-binary) [reply src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: But I remove the junk comments.

File: src/javascript/app_2/Stores/Modules/Trading/trade-store.js [hunk 2]
```diff
@@ -199,6 +200,36 @@ export default class TradeStore extends BaseStore {
...
+            if (current_active_symbol.exchange_is_open !== 1) {
+                const first_open_store = active_symbols.active_symbols.find(s => s.exchange_is_open === 1);
+                if (first_open_store !== undefined) {
```
[the remark this answers is not in the document]
Comment (arash-binary) [reply src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: Sure, I'm going to use a separate function for these lines.

File: src/javascript/app_2/Stores/Modules/Trading/trade-store.js [hunk 3]
```diff
@@ -199,6 +200,36 @@ export default class TradeStore extends BaseStore {
...
+                    URLHelper.setQueryParam({ 'symbol': first_open_store.symbol });
+                    query_string_values = this.updateQueryString();
+                }
+            }
+        } else {
+            // Have favorites
+            const clinet_first_open_symbol = active_symbols.active_symbols.find(symbol => client_favorite_store
```
[the remark this answers is not in the document]
Comment (arash-binary) [reply src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: Yes, the client's first open store from his/her favorites.

File: src/javascript/app_2/Stores/Modules/Trading/trade-store.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (OskarAhl) [reply src/javascript/app_2/Stores/Modules/Trading/trade-store.js]: Could be a boolean with `!!` but either way is fine I guess

**Pull request closed without merging**